metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -421,14 +429,22 @@ def _get_dataset_column_median(
         if not self.config.include_field_median_value:
             return
         try:
-            if self.dataset.engine.dialect.name.lower() == "snowflake":
+            if self.dataset.engine.dialect.name.lower() == SNOWFLAKE:
                 column_profile.median = str(
                     self.dataset.engine.execute(
                         sa.select([sa.func.median(sa.column(column))]).select_from(
                             self.dataset._table
                         )
                     ).scalar()
                 )
+            elif self.dataset.engine.dialect.name.lower() == BIGQUERY:
+                column_profile.median = str(
+                    self.dataset.engine.execute(
+                        sa.select(
+                            sa.text(f"approx_quantiles(`{column}`, 2) [OFFSET (1)]")
```

Comment:
we already have a patch for bq quantiles here https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py#L150 - is this also required?

Reply:
yes, as of version 0.15.50 of GX, median calculation does not use this patched method.

Reply:
could you add a comment explaining that

Reply:
can do it in a follow-up too